excel-importer/src/main/java/uk/gov/hmcts/ccd/definition/store/excel/parser/AuthorisationCaseFieldParser.java

```diff
@@ -47,14 +49,12 @@ void parseAndSetACLEntities(final Map<String, DefinitionSheet> definitionSheets,
             LOG.debug("Parsing AuthorisationCaseField for case type {}, caseField {}...",
                 caseTypeReference, caseFieldReference);
 
-            if (null == dataItems || dataItems.isEmpty()) {
+            if (dataItems.isEmpty()) {
```

Comment:
dataItems can no longer be null; it's now instantiated with getOrDefault() and will be an empty list if the lookup fails.

Reply:
Adding a null check returns a code smell from SonarCloud since `dataItems` is [called before](https://github.com/hmcts/ccd-definition-store-api/blob/fb3d49705db5ac5172c77bd6e5b1a1f4a0be7a01/excel-importer/src/main/java/uk/gov/hmcts/ccd/definition/store/excel/parser/AuthorisationCaseFieldParser.java#L43) without a visible null check.

Reply:
My bad, didn't notice the method change